pkg/cmd/convert.go

```diff
@@ -15,18 +15,25 @@ import (
 	"github.com/redhat-developer/opencompose/pkg/transform"
 	"github.com/redhat-developer/opencompose/pkg/transform/kubernetes"
 	"github.com/redhat-developer/opencompose/pkg/transform/openshift"
+	pkgutil "github.com/redhat-developer/opencompose/pkg/util"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"k8s.io/client-go/pkg/api"
 	"k8s.io/client-go/pkg/api/meta"
 	"k8s.io/client-go/pkg/runtime"
+	"net/url"
+	"time"
 )
 
 var (
 	convertExample = `  # Converts file
   opencompose convert -f opencompose.yaml`
 )
 
+const (
+	retryAttempts = 3
```

Comment:
I know it's not a C age, it's just that it becomes easier to understand just by looking at code that it's a constant and no need to find it's declaration in immediate scope.